website/content/docs/upgrading/deduplication/entity-group.mdx

```diff
@@ -0,0 +1,104 @@
+---
+layout: docs
+page_title: Fix entity and group duplicates
+description: >-
+  Fix duplicate identities for Vault entities and groups
+---
+
+# Fix entity and group duplicates
+
+    - [Fix ACL policy template duplicates](/vault/docs/upgrading/deduplication/acl-policy-templates)
+    - [Fix external reference duplicates](/vault/docs/upgrading/deduplication/external-refs)
+    
+### Resolving entity and group duplicates
+
+```
+[WARN]  identity: 2 entity duplicates found
+[WARN]  identity: entity "entity-cAsE" with namespace ID "root" duplicates 2 others: id=2562b42d-f603-ac6b-2591-8a20dd050897 force_deduplication="would rename to entity-cAsE-2562b42d-f603-ac6b-2591-8a20dd050897"
+[WARN]  identity: entity "entity-case" with namespace ID "root" duplicates 2 others: id=290a643d-6043-da5e-943f-3a3d09e4ecbd force_deduplication="would rename to entity-case-290a643d-6043-da5e-943f-3a3d09e4ecbd"
+[WARN]  identity: entity "entity-case" with namespace ID "root" duplicates 2 others: id=b0141be5-3f03-a1c7-a57b-02f045d04426 force_deduplication="would not rename"
+[WARN]  identity: entity "entity-cAsE" with namespace ID "sYMXY" duplicates 2 others: id=95f0743b-a1d5-26da-b4ef-a50490da0787 force_deduplication="would rename to entity-cAsE-95f0743b-a1d5-26da-b4ef-a50490da0787"
+[WARN]  identity: entity "entity-case" with namespace ID "sYMXY" duplicates 2 others: id=9d3be96f-490a-9625-3118-896bd2b3a5f3 force_deduplication="would rename to entity-case-9d3be96f-490a-9625-3118-896bd2b3a5f3"
+[WARN]  identity: entity "entity-case" with namespace ID "sYMXY" duplicates 2 others: id=d82231d2-3716-6b9c-e80d-7d09c0409739 force_deduplication="would not rename"
+[WARN]  identity: end of entity duplicates
+[WARN]  identity: 2 group duplicates found
+[WARN]  identity: 2 group duplicates found
+[WARN]  identity: group "group-case" with namespace ID "root" duplicates 2 others: id=8ad26e0c-8cf6-5b67-7c77-6571fa374f34 force_deduplication="would rename to group-case-8ad26e0c-8cf6-5b67-7c77-6571fa374f34"
+[WARN]  identity: group "group-cAsE" with namespace ID "root" duplicates 2 others: id=9fe86ea0-f80c-1131-5be1-1d6e3b70237f force_deduplication="would rename to group-cAsE-9fe86ea0-f80c-1131-5be1-1d6e3b70237f"
+[WARN]  identity: group "group-case" with namespace ID "root" duplicates 2 others: id=32dd070c-c1f8-c796-9a71-15887014b813 force_deduplication="would not rename"
```

Comment:
Technically we changed which of these say "would not rename" - it would now be the first one not the last. But I don't think that matters. for the docs here?

Reply:
It's always better if the examples in the dogs match the actual output. I'll fix it.